flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/TableEnvironmentITCase.scala

```diff
@@ -808,6 +808,162 @@ class TableEnvironmentITCase(tableEnvName: String, isStreaming: Boolean) extends
     listener.close()
   }
 
+  // `SHOW COLUMNS ......` syntax of flink SQL test cases.
+  @Test
+  def testShowColumnsSyntaxOfSQL(): Unit = {
```

Comment:
This test is a little verbose and we have many more similar statements to test, actually, they dont' have much thing to do with `TableEnvironment`. Could you extract the test into a separate test class, e.g. `ExecuteSqlTest`.

Reply:
What about adding the  `ExecuteSqlTest` in the scala package `org.apache.flink.table.api` of module `flink-table-planner` ?

Reply:
Sounds good to me.